src/components/skeleton/Header/Header.js

```diff
@@ -28,7 +29,9 @@ export default class Header extends Component {
     const t = this.context.t;
     const loginOrProfile = !this.props.user.playerId ?
       <FlatButton label={t('nav.login')} onClick={() => browserHistory.push(t.route('login'))} /> :
-      <Link className="Header-link Header-icon-right" to={t.route('profile')}><Icon fa="fa fa-2x fa-user" /></Link>;
+      <Link className="Header-link Header-icon-right" to={t.route('profile')}>
+          <Icon fa="fa fa-2x fa-user" />&nbsp;&nbsp;<b>{this._reverseName(storeUtil.getPlayer(this.props.user.playerId).name)}</b>
+      </Link>;
```

Comment:
Zoizo moet er nog tekst vanboven komen. Logo moet nog veranderen enzo.

Functioneel gezien:

- Nu kan een persoon zien met welke gebruiker hij ingelogd is (Men kan ook inloggen met een medespeler als men zijn/haar paswoord weet)
- Nu weet ge dat ge met de juiste persoon bezig zijt (bv. Dirk DS weet dat hij bezig is met Dirk DS en niet Dirk B)

-> Ik zou het dus zeker erin stoppen als is het maar voor een gebruiker info te geven